etc/systemd/system/sidekiq-prometheus-exporter.service

```diff
@@ -3,10 +3,48 @@ Description=Sidekiq Prometheus Exporter
 After=network.target
 
 [Service]
+User=mastodon
 WorkingDirectory=/var/lib/sidekiq-prometheus-exporter
-ExecStart=bundle exec rackup -p31337 -o0.0.0.0
+Environment="PUMA_ENV=production"
+Environment="BIND=0.0.0.0"
+Environment="PORT=31337"
```

Comment:
@Preskton we should probably make these set via an override file. I'm not sure why we need this listening on 0.0.0.0 though?

Reply:
Yup, on the host we are binding to localhost.